pkg/utils/configMapUtils.go

```diff
@@ -93,7 +87,7 @@ func getEngineConfigmaps(clients ClientSets, engineDetails EngineDetails, expDet
 }
 
 // OverridingConfigMaps will override configmaps from ChaosEngine
-func OverridingConfigMaps(experimentConfigMaps []v1alpha1.ConfigMap, engineConfigMaps []v1alpha1.ConfigMap, expDetails *ExperimentDetails) {
+func (expDetails *ExperimentDetails) OverridingConfigMaps(experimentConfigMaps []v1alpha1.ConfigMap, engineConfigMaps []v1alpha1.ConfigMap) {
```

Comment:
To make it more explicit, can we name this function as `getOverridingConfigMapsFromChaosEngine`